presentation/src/main/java/team/jsv/icec/ui/camera/CameraActivity.kt

```diff
@@ -13,4 +19,40 @@ class CameraActivity : BaseActivity<ActivityCameraBinding>(R.layout.activity_cam
         super.onCreate(savedInstanceState)
         requestPermissions(PermissionUtil.getPermissions())
     }
+
+
+    override fun onWindowFocusChanged(hasFocus: Boolean) {
+        super.onWindowFocusChanged(hasFocus)
+        hideSystemUI()
+    }
+
+    private fun hideSystemUI() {
+        if (Build.VERSION.SDK_INT >= VERSION_CODES.R) {
+            window.setDecorFitsSystemWindows(false)
+            window.insetsController?.hide(WindowInsets.Type.statusBars())
+            setBottomMargin()
+        } else {
+            window.decorView.systemUiVisibility =
```

Comment:
```suggestion
        } else {
            @Suppress("DEPRECATION") // (@jiiiiiyoon) : ~~~
            window.decorView.systemUiVisibility =
```

- `else` 블럭에서 `Deprecated` 경고가 발생을 하는데 해당 부분의 사용이 필수이므로 `else` 블럭안에 `@Suppress("DEPRECATION")`를 작성하고 어노테이션을 작성해준 이유와 깃허브 아이디정도 같이 놓아주면 더 좋을것 같아요~!

Reply:
수정 완~